x-pack/plugins/cross_cluster_replication/public/app/sections/auto_follow_pattern_add/auto_follow_pattern_add.js

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { PureComponent, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { injectI18n, FormattedMessage } from '@kbn/i18n/react';
+
+import {
+  EuiBreadcrumbs,
+  EuiPage,
+  EuiPageBody,
+  EuiPageContent,
+  EuiPageContentHeader,
+  EuiSpacer,
+  EuiTitle,
+  EuiButton,
+  EuiCallOut,
+} from '@elastic/eui';
+
+import routing from '../../services/routing';
+import { BASE_PATH_REMOTE_CLUSTERS } from '../../../../common/constants';
+import { AutoFollowPatternForm, RemoteClustersProvider, SectionLoading, SectionError } from '../../components';
+
+class AutoFollowPatternAddUI extends PureComponent {
```

Comment:
Minor nit, but I think `UI` should be `Ui` to follow the camel case pattern.

Reply:
👍